### ISSUE TITLE
Compatibility with Netbeans 8?

### ISSUE THREAD

**Issue by jmansion** [dcaoyuan/nbsbt#10]
I have a project that's working OK with nbeclipse.  I install the 1.1.2 plugin as directed (sbt 0.13.5 and Scala 2.11.1) and go 'netbeans'.  As expected it creates .classpath_nb.

However, Netbeans 8.0 is only offering to open 'Project Folder' and navigating to the directory containing .classpath_nb does not work as a suitable directory it seems.

I believe I have all the scala plugins from the NB plugins portal installed.

![image](https://cloud.githubusercontent.com/assets/1613161/3461481/760a0c22-0223-11e4-851e-493083300ee1.png)

I'm a Netbeans noob, so its quite possibly my fault.

**Comment by jasongoodwin** [comment]
Same issue observed here. Also a netbeans noob (writing a book so trying to include instructions for netbeans)

**Comment by mgenereu** [comment]
Same question. Do you have a build.sbt in that folder?

**Comment by jasongoodwin** [comment]
Yes. Project is the activator minimal Java or scala akka seed project. I also tried generating the project files with activator net beans. No luck. If I create an sbt project in net beans I can open it though.

**Comment by jasongoodwin** [comment]
Here is the section on opening the project. Maybe you can review and let me know if something is wrong? Or otherwise provide input? Thanks in advance for any insight.

Netbeans
Here we will install Netbeans 7.4 and Sbt and Scala integration.
This is current as of the time of writing however this could change over time. Netbeans support sbt and scala out of the box so these instructions may become simpler over time.

Download Netbeans SE from https://netbeans.org/downloads
Install by following prompts in installer.

Open Netbeans once installation is complete.

Goto Tools -> Plugins
Click on "Available Plugins" tab.
Click the "Check for Newest" button.
Type "scala" in the Search field.
Select the Scala SBT Project plugin and Scala plugin. You may want the Scala Refactoring plugin as well.

netbeansinstall.png

Click install (it will also list several dependencies).
Follow the prompts to instal all of the plugins.

Preparing the Project for Netbeans

Open your global sbt plugins file (create it if it's not there) - it is located in ~/.sbt/{version}/plugins/build.sbt where version is the sbt version. It is 0.13 at the time of writing.

Include the following in the file ensuring there is a blank line between each line in the file:

addSbtPlugin("org.netbeans.nbsbt" % "nbsbt-plugin" % "1.1.4")

*It's possible that the current version of the nbsbt-plugin has changed since the time of writing - check https://github.com/dcaoyuan/nbsbt for any updates.

Once you have done this, you'll be able to generate the project files by navigating to the sbt project root we generated earlies (akkademy-db).

Once you're in the project folder's root, run 'activator netbeans'
